module.py

```diff
@@ -551,6 +551,8 @@ def handle_op(self, codeobj, context, label, op, oparg, line):
             context.end_block()
 
         elif op == BUILD_CONST_KEY_MAP:
+           context.add_decl_once('i', 'int', None, False)
+
```

Comment:
This is missing a space for its indentation.

Reply:
Fixed, thanks for noticing.